packages/wrangler/src/bundle-reporter.tsx

```diff
@@ -0,0 +1,42 @@
+import { readdirSync, statSync } from "fs";
+import { join } from "node:path";
+import { gzipSync } from "node:zlib";
+import { Box, Text, render, useStdout } from "ink";
+import * as React from "react";
+import type { DirectoryResult } from "tmp-promise";
+
+const dirSize = (dirPath: string) =>
+  readdirSync(dirPath)
+    .map((file) => statSync(join(dirPath, file)))
+    .reduce((acc, { size }) => acc + size, 0);
+
+function getCompressedSize(size: string): string {
+  return `${(dirSize(size) / 1024).toFixed(2)} KiB / gzip: ${(
+    gzipSync(size).length / 1024
+  ).toFixed(2)} KiB`;
+}
+
+export function BundleReporter({
+  tempDirectory,
+}: {
+  tempDirectory: DirectoryResult;
+}) {
+  const { unmount } = render(<BundleSize tempDirectory={tempDirectory} />);
+
+  unmount();
+}
+
+function BundleSize({ tempDirectory }: { tempDirectory: DirectoryResult }) {
```

Comment:
let's not use ink for this and just log it out. it probably shouldn't be a big box either, just a single line with the details should do.